amida-tech/indaba-client
# Feature/inba 940 check question completion

#### What does this PR do?

Survey used to send toasts if a section name or question was not filled out when hitting save. 

Now, when moving from Draft -> Published these same checks are done. if anything is blank a toast appears. 
Section names are also autofilled if a user hits save without updating the section name. 

#### Related JIRA tickets:

[INBA-940](https://jira.amida.com/browse/INBA-940)

#### How should this be manually tested?

Create a survey that is not completely filled out in the project wizard and the normal place. 
Try to change to draft and see errors.

#### Background/Context

#### Screenshots (if appropriate):
